linera-execution/src/execution.rs

```diff
@@ -321,6 +322,45 @@ where
         }
     }
 
+    pub async fn bounce_message(
+        &self,
+        context: &MessageContext,
+        message: Message,
+    ) -> Result<Vec<ExecutionResult>, ExecutionError> {
+        assert_eq!(context.chain_id, self.context().extra().chain_id());
+        match message {
+            Message::System(message) => {
+                let mut result = RawExecutionResult {
+                    authenticated_signer: context.authenticated_signer,
+                    ..Default::default()
+                };
+                result.messages.push(RawOutgoingMessage {
+                    destination: Destination::Recipient(context.message_id.chain_id),
+                    authenticated: true,
```

Comment:
Would it be safer to say `false`?
I guess `true` is not necessarily dangerous; it's just important to keep in mind (also for app developers, for the user messages below?) that this means the _bouncing_ is authenticated by the sender (the bouncing chain), not the message itself.

Reply:
No, the bouncing will be authenticated by the original sender.